install_config/storage_examples/gluster_backed_registry.adoc

```diff
@@ -13,27 +14,18 @@ toc::[]
 
 == Overview
 
-This topic reviews how to attach a GlusterFS persistent volume to the Docker
-Registry.
+This topic reviews how to attach a GlusterFS volume to an integrated OpenShift
+Container Registry. This can be done with any of {gluster-native},
+{gluster-external}, or standalone {gluster}. It is assumed that the registry
+has already been started and a volume has been created.
 
-It is assumed that the Docker registry service has already been started and the
-Gluster volume has been created.
-
-[[backing-docker-registry-with-glusterfs-storage-prerequisites]]
+[[prerequisites]]
```

Comment:
@jarrpa - this anchor tag is used twice in this PR. Can you change this to be more specific to this topic?